frontend/src/components/Contexts/SignalRContext.tsx

```diff
@@ -81,7 +81,14 @@ export const SignalRProvider: FC<Props> = ({ children }) => {
     }, [accessToken])
 
     const registerEvent = (eventName: string, onMessageReceived: (username: string, message: string) => void) => {
-        if (connection) connection.on(eventName, (username, message) => onMessageReceived(username, message))
+        if (connection)
+            connection.on(eventName, (username, message) => {
+                if (message === 'null') {
```

Comment:
Is the message a string with just "null" in it, and not a null string?

Reply:
Yes, the message is 'null' and its only after JSON.parse that we end up with null